### DISCUSSION TITLE
I Can't install the pybamm in window 11 , python version 3.11.2

### DISCUSSION THREAD

**Discussion by mmpoom** [pybamm-team/PyBaMM#2854] [Q&A] [answered]
When I try to install PyBaMM on Windows 11 with python version 3.11.2. It have ERROR: ResolutionImpossible problem. How can I fix this? 

Problem ::

ERROR: Cannot install pybamm==0.1.0, pybamm==0.2.0, pybamm==0.2.1, pybamm==0.2.2, pybamm==0.2.2.post1, pybamm==0.2.2.post2, pybamm==0.2.2.post3, pybamm==0.2.3, pybamm==0.2.3.post1, pybamm==0.2.4, pybamm==0.2.4.post1, pybamm==0.2.4.post2 and pybamm==0.2.4.post3 because these package versions have conflicting dependencies.

The conflict is caused by:
    pybamm 0.2.4.post3 depends on casadi>=3.5.0
    pybamm 0.2.4.post2 depends on casadi>=3.5.0
    pybamm 0.2.4.post1 depends on casadi>=3.5.0
    pybamm 0.2.4 depends on casadi>=3.5.0
    pybamm 0.2.3.post1 depends on casadi>=3.5.0
    pybamm 0.2.3 depends on casadi>=3.5.0
    pybamm 0.2.2.post3 depends on casadi>=3.5.0
    pybamm 0.2.2.post2 depends on casadi>=3.5.0
    pybamm 0.2.2.post1 depends on casadi>=3.5.0
    pybamm 0.2.2 depends on casadi>=3.5.0
    pybamm 0.2.1 depends on casadi>=3.5.0
    pybamm 0.2.0 depends on casadi>=3.5.0
    pybamm 0.1.0 depends on casadi>=3.5.0

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts

**Comment by Saransh-cpp** [comment, marked as answer]
Hi @mmpoom, pybamm is currently not available on Python 3.11. Please install pybamm in a Python 3.8 or 3.9 environment.

**Reply by mmpoom** [reply]
Thankyou, now I can install and use PyBamm on Python3.9

**Reply by mmpoom** [reply]
I have another question about Pybamm Li-ion model. Can Pybamm simulate the LiFePO4 cathode li-ion battery? and what the chemicals have in pybamm? Thankyou
